projects/js-toolkit/packages/liferay-cli/src/new/index.ts

```diff
@@ -120,5 +119,12 @@ export default async function newProject(
 }
 
 function loadTargets(): Target[] {
-	return [targetLiferay];
+	return (
+		fs
+			.readdirSync(__dirname, {withFileTypes: true})
+			.filter((dirent) => dirent.isDirectory())
+			.filter((dirent) => dirent.name.startsWith('target-'))
```

Comment:
How are we going to generate the target platforms?

Reply:
https://github.com/liferay/liferay-frontend-projects/pull/546, this PR contains the script for generating new platforms.

Reply:
❤️ So many PRs to read :(

Reply:
![](https://media.giphy.com/media/l4EoOFliX6Vw4PD20/giphy.gif)

Reply:
About this ^ I expect Brian wanting to have a version of publishing the target platform automatically, either invoked by a release script or integrated somewhere.

Reply:
I'm thinking that we may need another package with the common code for target platforms and then, make the script only generate the portal dependencies. This is because we will need code in addition to `build` to, for example, `deploy`. That will make it easier for Brian to publish it.

I will give it a thought.

Reply:
Created [this issue](https://github.com/liferay/liferay-frontend-projects/issues/591) to reflect on this.